idb/postgres/postgres.go

```diff
@@ -2092,6 +2095,9 @@ func (db *IndexerDb) yieldAccountsThread(req *getAccountsRequest) {
 			*account.CreatedAssets = cal
 		}
 
+		var totalSchema models.ApplicationStateSchema
+		var totalExtraPages uint64
```

Comment:
nit: move this into the `len(appParamIndexes) > 0` block.

Reply:
Only `totalExtraPages` can be moved - done. `totalSchema` is calculated for both created and opted-in apps